pkg/logs/client/destination.go

```diff
@@ -74,3 +85,35 @@ func (d *Destination) Send(payload []byte) error {
 
 	return nil
 }
+
+// SendAsync sends a message to the destination without blocking. If the queue is full, the incoming messages will be
+// dropped
+func (d *Destination) SendAsync(payload []byte) {
```

Comment:
Should we create a `AsyncDestination` ?

Reply:
That would probably imply creating an interface and duplicating a lot of code. The logic is really tied with the `Main` and `Additional` endpoints, so the boundary is quite clear. I don't see any other usage of this in the forseeable future that would justify it

Reply:
Not sure about this, `AsyncDestination` could just be a wrapper of `Destination`